var/spack/repos/builtin/packages/py-neuron-reduce/package.py

```diff
@@ -11,9 +11,10 @@ class PyNeuronReduce(PythonPackage):
 
     homepage = "https://github.com/orena1/neuron_reduce"
     url = "https://pypi.io/packages/source/n/neuron_reduce/neuron_reduce-0.0.6.tar.gz"
-    git = "git@github.com:orena1/neuron_reduce.git"
+    git = "git@github.com:BlueBrain/neuron_reduce.git"
 
     version('develop', branch='master')
+    version('0.0.9', commit='b8738a17f0234ddc6470108c1f84fc8b27c4cc0f')
```

Comment:
nitpick : as we have our own repo now (originally it was under Oren's account only), we should start using tags for new versions.